src/scaffolding/extrinsicHelpers.ts

```diff
@@ -23,51 +21,54 @@ import type { Call } from '@polkadot/types/interfaces/runtime';
 import { KeyringPair } from '@polkadot/keyring/types';
 import util from 'util';
 import { connect, connectPromise } from './apiConnection.js';
-// eslint-disable-next-line import/no-cycle
 import { getBlockNumber, log, getDefaultFundingSource, Sr25519Signature, EXISTENTIAL_DEPOSIT } from './helpers.js';
 import { ModelTypeStr, PayloadLocationStr, SchemaSettingStr } from './schema.js';
 
-export type ReleaseSchedule = {
+export interface ReleaseSchedule {
```

Comment:
Just curious; any particular functional reason for all the `type` -> `interface` changes?